### PR TITLE
chore: prepare for 2.14/individually versioned component release

### PR DESCRIPTION
**Pull request by lazd**
## Description

This PR updates peer dependencies and makes things ready for a public release, addressing SDS-2443

## How and where has this been tested?
 - **How this was tested:** local release against Verdaccio
 - **Browser(s) and OS(s) this was tested with:** n/a

## To-do list

- [x] If my change impacts other components, I have tested to make sure they don't break.
- [x] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [x] This pull request is ready to merge.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,9 +16,8 @@
     "backstop:clean": "rm -rf backstop_data/bitmaps_test backstop_data/html_report",
     "backstop:docker:test": "backstop test --docker --config=backstop_data/backstop_test.js --env=local",
     "backstop:ci:test": "backstop test --docker --config=backstop_data/backstop_test.js --env=ci",
+    "backstop:test-all": "backstop test --docker --config=backstop_data/backstop_test.js --env=local themes=lightest,light,dark,darkest scales=medium,large",
```
Comment (bernhard-adobe) [inline package.json]: this PR might need a rebase @lazd . The rest of the PR looks great.
- removed alphas and updated script to do semver automatically for packages, right?

Comment (lazd) [reply package.json]: Looks like that was just renaming the `prepublishOnly` script. We're good!

File: package.json [hunk 2]
```diff
@@ -70,5 +69,8 @@
     "hooks": {
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
+  },
+  "dependencies": {
+    "semver": "^6.3.0"
```
Comment (jianliao) [inline package.json]: Should this dependency be moved to devDependencies?

Comment (lazd) [reply package.json]: yes!

**Review by yvettep321 — COMMENTED** [review]
thank you

**Pull request merged**